code/game/objects/items/storage/pill_packets.dm

```diff
@@ -11,16 +11,26 @@
 
 /obj/item/storage/pill_bottle/packet/tricordrazine
 	name = "Tricordazine pill packet"
-	desc = "This packet containts tricordazine pills. Heals all types of damage minorly. Once you take them out they don't go back in. No more than 2 pills in a short period."
+	desc = "This packet containts tricordazine pills. Heals all types of damage slightly. Once you take them out they don't go back in. No more than 2 pills in a short period."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/tricordrazine
 
 /obj/item/storage/pill_bottle/packet/paracetamol
 	name = "Paracematol pill packet"
 	desc = "This packet containts paracetamol pills, also known as tylenol. A long lasting but minor painkiller. Once you take them out they don't go back in. No more than 4 pills in a long period."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/paracetamol
 
-
 /obj/item/storage/pill_bottle/packet/leporazine
 	name = "Leporazine pill packet"
 	desc = "This packet containts leporazine pills. Rapidly stablizes the patients internal temperature. Once you take them out they don't go back in.."
 	pill_type_to_fill = /obj/item/reagent_containers/pill/leporazine
+
+/obj/item/storage/pill_bottle/packet/russianRed
```

Comment:
Snake case

Reply:
Uses the same case as the pre-existing chem, pill, and pill bottle for consistency. I can change the things I added if you really want me to, but that seems awkward.

Reply:
snake case

Reply:
Yes, that's a statement. Do you want snake case, or do you want me to use a different typepath than the other things?

Reply:
```suggestion
/obj/item/storage/pill_bottle/packet/russian_red
```